terraform/modules/tf_lambda/variables.tf

```diff
@@ -68,29 +73,39 @@ variable "auto_publish_versions" {
   description = "Whether Terraform should automatically publish new versions of the function"
 }
 
+variable "alias_name" {
+  default     = "production"
+  description = "An alias with this name is automatically created which points to aliased_version"
+}
+
 variable "aliased_version" {
   default     = ""
   description = "Alias points to this version (or the latest published version if not specified)"
 }
 
+variable "invocation_frequency_minutes" {
```

Comment:
Could we change this to just accept either a `rate(...)` or a cron based expression?  That way, users have the freedom to use hours, days, etc.

Reply:
Good idea - I was thinking about this, too. The more generalized form would also allow the `apps` to migrate to this module